src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/README.md

```diff
@@ -0,0 +1,163 @@
+# GraphQL API
+
+## Query
+Every GraphQL schema has a root type for both queries and mutations. The query type defines GraphQL operations that retrieve data from the server.
+
+### contentItem
+The following query looks up a content item on the contentItemId `A913LFSOAQWEM`, the query then returns the `contentItemId` and `contentType`
+
+```graphql
+query {
+  contentItem(contentItemId:"A913LFSOAQWEM") {
+    contentItemId
+    contentType
+  }
+}
+```
+
+The schema at this point doesnt know about contentparts. This query is for just the raw content item.
+
+### contentItems
+Similar to the previous query, this query will return a list of content items, rather than a singular one.
+
+This query will return all the `contentItemId` and `contentType` values of content with a content type of `BlogPost`.
+
+```graphql
+query {
+  contentItems(contentType: "BlogPost") {
+    contentItemId
+    contentType
+  }
+}
+```
+
+### Dynamic Querying
+Because Orchard has a very fluid content model, i.e. You can create content types on the fly, and these are not represented in code, content types are dynamically built.
+
+An example is, lets say a `Blog` content type exists within the content type system, here you can query all blogs.
+
+```graphql
+query {
+  blog(id: "D394KFSDERFDM") {
+    contentItemId
+    contentType
+  }
+}
+```
+
+This query will return a list of all blog's `contentItemId` and `contentType` properties. These endpoints do not have a singular representation.
+
+#### Query, returning content part details
+All content items contain content parts, and once you have a content item, you will more than likely want details of, lets say the title on the titlePart.
+
+So lets return a Blog, with an Id of "D394KFSDERFDM", and display its Title contained on the TitlePart.
+
+```graphql
+query {
+  blog(id: "D394KFSDERFDM") {
+    contentItemId
+    contentType
+    titlePart {
```

Comment:
Bot such a good example as the title part 's value will be deprecated, and we should use DisplayText instead. Maybe better to use MarkdownPart in this case instead.